### DISCUSSION TITLE
defroum extension generate blur

### DISCUSSION THREAD

**Discussion by adird3** [lshqqytiger/stable-diffusion-webui-directml#141] [Q&A]
Hi, i have windows 10 with rx 5500xt
i have tried to use deforum with the stable-diffusion-webui-directml.
in 2D mode it works fine, but when i am trying to use the 3D mode with init photo the output of the first photo is good, but all the rest photo that come after is just in one color and any configuration change that i made it is stay the same
anyone else have this problem?

this is example of the output:
![20230529183133_000000001](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/133755492/359911c6-fb0c-440d-a534-d50866dc9f73)

**Comment by mercurius08** [comment]
Could you solve it? I have a GTX6800xt and I have the same problem.

**Comment by adird3** [comment]
Unfortunately not, still with the same problem

**Comment by pw405** [comment]
The GitHub pages states it is specifically designed to use on Google Colab (super computers) and has some specific CUDA requirements.  It may not work with the DirectML version of SD that we're using.  

You could possibly try to tinker with command line arguments, etc. 

When the blurry image generates, is it outputting any errors in the command prompt console?

**Reply by mercurius08** [reply]
no, it makes the video as if it were making it normal... but when it enters the folder you only see the first correct image and the rest are all blurred.

**Reply by mercurius08** [reply]
example: 

https://github.com/lshqqytiger/stable-diffusion-webui-directml/issues/29#issuecomment-1685175265
